tests/_runtime/test_dataflow_cases.py

```diff
@@ -306,9 +306,6 @@ def __post_init__(self) -> None:
         },
         # TODO: What should the defs be?
```

Comment:
can we remove this TODO?

Reply:
there are a few other comments either from @Light2Dark or me `# expected_refs`. do you mind cleaning those up and verifying they look correct to you